aspnetcore/tutorials/getting-started-with-swashbuckle.md

```diff
@@ -91,6 +91,8 @@ In the `Startup.Configure` method, enable the middleware for serving the generat
 
 [!code-csharp[](../tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.Swashbuckle/Startup2.cs?name=snippet_Configure&highlight=4,8-11)]
 
+The preceding `UseSwaggerUI` method call enables the [static files middleware](xref:fundamentals/static-files). If targeting .NET Framework or .NET Core 1.x, add the [Microsoft.AspNetCore.StaticFiles](https://www.nuget.org/packages/Microsoft.AspNetCore.StaticFiles/) NuGet package to the project.
```

Comment:
* When named explicitly, we've been making built-in middleware names proper nouns ... "Static Files Middleware"
* This might be fine (".NET Core" vs. "ASP.NET Core"), but haven't we been consistently using "ASP.NET Core"?

Reply:
@guardrex Not sure what you mean in the 2nd bullet point. Please clarify.

Reply:
> If targeting .NET Framework or .NET Core 1.x,

I usually see (and use myself "ASP.NET Core 1.x") for these references, so I was just wondering if it's ok as is or if it should go ...

> If targeting .NET Framework or ASP.NET Core 1.x,

Reply:
I prefer ".NET Framework or .NET Core 1.x", since they're both target frameworks. This makes the statement more symmetrical.